MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1417,39 +1413,25 @@ private static Video[] LoadThemeVideos(IEnumerable<FileSystemMetadata> fileSyste
 
         protected virtual BaseItem[] LoadExtras(List<FileSystemMetadata> fileSystemChildren, IDirectoryService directoryService)
         {
-            var extras = new List<Video>();
-
-            var libraryOptions = new LibraryOptions();
-            var folders = fileSystemChildren.Where(i => i.IsDirectory).ToList();
-            foreach (var extraFolderName in AllExtrasTypesFolderNames)
-            {
-                var files = folders
-                    .Where(i => string.Equals(i.Name, extraFolderName, StringComparison.OrdinalIgnoreCase))
-                    .SelectMany(i => FileSystem.GetFiles(i.FullName));
-
-                // Re-using the same instance of LibraryOptions since it looks like it's never being altered.
-                extras.AddRange(LibraryManager.ResolvePaths(files, directoryService, null, libraryOptions)
+            return fileSystemChildren
+                .Where(child => child.IsDirectory)
+                .Where(folder => AllExtrasTypesFolderNames.ContainsKey(folder.Name))
+                .SelectMany(folder => LibraryManager
+                    .ResolvePaths(FileSystem.GetFiles(folder.FullName), directoryService, null, new LibraryOptions())
```

Comment:
I don't know if it's needed, but `LibraryOptions` can be retrieved from `LibraryManager.GetLibraryOptions(BaseItem)`

Reply:
I don't know either, so I kept the original code.